levenshtein/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+from levenshtein.models import Levenshtein
+from levenshtein.forms import LevenshteinForm
+
+def index(request):
+
+    levenshtein_objects = Levenshtein.objects.all()
+    
+    if request.method == 'POST':
+        form = LevenshteinForm(request.POST)
+        if form.is_valid():
+            levenshtein = form.save()
+            levenshtein.levenshtein_value = levenshteinDistance(levenshtein.text_1, levenshtein.text_2)
+            levenshtein.save()
+    else:
+        form = LevenshteinForm()
+
+    return render(request, 'levenshtein/index.html', {'form': form, 'levenshtein_objects':levenshtein_objects})
+
+def levenshteinDistance(s1,s2):
```

Comment:
Also `(s1,s2)` should be turned into `(s1, s2)`. Please enable a PEP8 linter for your IDE/Editor, so you can get help to catch this things more easily.